package.json

```diff
@@ -42,9 +42,7 @@
     "async": "2.4.0",
```

Comment:
Ya, removing the caret (or tilde) which would allow other versions: https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json